.agent/memory.md

```
@@ -335,4 +335,10 @@ COMPOSE_PROJECT_NAME=coneshare docker-compose exec frontend npm test -- --run sr
 - **Context/Implication:** Removing or renaming documents and folders in a Dataroom broke the View Sessions activity log (rendering empty "Viewed document: " strings or retroactively mutating historical names).
 - **Resolution/Action:** Added point-in-time snapshot fields (`item_type`, `item_name`, `item_path`, `document_type`) to `DataroomVisit` populated at access/download time, with `get_full_path()` hierarchy tracking on `DataroomFolder`. `DataroomVisitSerializer` falls back to snapshots when foreign keys are null and outputs `item_status` (`active`, `deleted`, `renamed`). The frontend renders subtle `[Deleted]` and `[Renamed]` status badges and path tooltips. Data backfill migration `0010` backfills existing live rows and omits `elidable=True` so squashed migrations preserve the backfill for unmigrated environments.
 
+### 2026-09-05 Session Entry
```

markdownlint-cli2 (0.23.2)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)